StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet.swift

```diff
@@ -315,11 +315,22 @@ extension PaymentSheet: PaymentSheetViewControllerDelegate {
     }
 
     func paymentSheetViewControllerDidSelectPayWithLink(_ paymentSheetViewController: PaymentSheetViewControllerProtocol) {
-        self.presentPayWithLinkController(
-            from: paymentSheetViewController,
-            intent: paymentSheetViewController.intent,
-            elementsSession: paymentSheetViewController.elementsSession
-        )
+        if configuration.forceEnableNativeLink {
+            self.presentPayWithNativeLinkController(
+                from: paymentSheetViewController,
+                intent: paymentSheetViewController.intent,
+                elementsSession: paymentSheetViewController.elementsSession,
+                shouldOfferApplePay: false,
+                shouldFinishOnClose: true,
```

Comment:
`presentPayWithNativeLinkController` is only being called here. There doesn't seem to be a need to pass in `shouldOfferApplePay` nor `shouldFinishOnClose`

Reply:
I'm working on another PR right now that sets these to the opposite values — if we're launching directly into Link before PS, then we do want to offer Apple Pay.